fix: add description for pages where the first line uses JS component (#327)
openfga/openfga.dev: docs/content/modeling/roles-and-permissions.mdx

---
sidebar_position: 5
slug: /modeling/roles-and-permissions

> description: Model roles and permissions using the authorization model and relationship tuple.

Reviewer: ```suggestion
description: Modeling basic roles and permissions in OpenFGA
```
